src/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -305,7 +306,6 @@ public boolean onTouch(View v, MotionEvent event) {
     container.setOnTouchListener(swipeListener);
 
   }
-
```

Comment:
Removed for what reason?

Reply:
Will revert, don't remember why. Will remove line 306.